app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError)

```

Comment:
read the checklist and fix the order of imports
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.